src/site/content/en/fast/replace-gifs-with-videos/index.md

```diff
@@ -54,6 +54,12 @@ ffmpeg -i my-animation.gif -b:v 0 -crf 25 -f mp4 -vcodec libx264 -pix_fmt yuv420
 This tells FFmpeg to take `my-animation.gif` as the **input**, signified by the
 `-i` flag, and to convert it to a video called `my-animation.mp4`.
 
+The libx264 encoder only works with files that have even dimensions, like 320x240px. If the input GIF has odd dimensions we can include a crop filter to avoid FFmpeg throwing a 'height/width not divisible by 2' error:
```

Comment:
```suggestion
The libx264 encoder only works with files that have even dimensions, like 320px by 240px. If the input GIF has odd dimensions you can include a crop filter to avoid FFmpeg throwing a 'height/width not divisible by 2' error:
```